cloud/ec2_util.go

```diff
@@ -271,6 +273,7 @@ func makeBlockDeviceMappings(mounts []MountPoint) ([]*ec2aws.BlockDeviceMapping,
 			m.Ebs = &ec2aws.EbsBlockDevice{
 				DeleteOnTermination: aws.Bool(true),
 				VolumeSize:          aws.Int64(mount.Size),
+				VolumeType:          aws.String(mount.VolumeType),
```

Comment:
You probably want to only set this if the user sets it. Otherwise leave it a nil pointer.